test/migration.test.js

```diff
@@ -246,6 +249,9 @@ describe('migrations', function() {
   });
 
   it('should autoupdate', function(done) {
+    if (mysql56) {
```

Comment:
in fact, I prefer if you rename the file to something more obvious like `var mySqlVersion = require('../helper/mysql-version');`, then you can `if (mysqlVersion.is('56')) {` ...